packages/ai/src/ButtonState.ts

```diff
@@ -67,6 +67,15 @@ class ButtonState extends UI5Element {
 	 */
 	@property()
 	endIcon?: string;
+
+	/**
+	 * Defines if the component is in split button mode.
+	 * @default false
+	 * @since 2.6.0
+	 * @public
+	 */
+	@property({ type: Boolean })
+	splitMode = false;
```

Comment:
I believe this should be called `showArrowButton` because `splitMode` means nothing for people who haven't seen the specs/figmas and is more like designer slang IMO